prov/efa/src/rdm/rxr_pkt_cmd.c

```diff
@@ -524,12 +524,14 @@ ssize_t rxr_pkt_trigger_handshake(struct rxr_ep *ep,
 	    (peer->flags & EFA_RDM_PEER_REQ_SENT))
 		return 0;
 
+	/* TODO: use rxr_ep_alloc_tx_entry to allocate tx_entry */
```

Comment:
Could you explain why this is not done currently?

Reply:
rxr_ep_alloc_tx_entry() require passing a `fi_msg`, which require constructing a `fi_msg`